LearnositySDK/LearnositySDK.csproj

```diff
@@ -1,83 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{4A1F8BE2-02A2-424D-AE33-11F742E18EED}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>LearnositySDK</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
@gonzalozawa what did you pick the version of .NET standard based on? They recommend:
> to target the lowest version of .NET Standard possible

Reply:
@karoltarasiuk this is a great question. I've tried using a lower version of .NET Standard but the project doesn't compile and throw errors. Mainly on Remote.cs and Init.cs
The problem in that particular classes is that System.Web and System.Net aren't included in the previous version of .NET Standard. See https://stackoverflow.com/questions/30227219/how-to-use-webclient-with-netcore
Do you think I should try to fix those classes and use .NET Core functionality or should we just target .NET Standard 2.0 ?

Reply:
Cool, makes sense. I think that in this case we leave it as is and only come back to it if somebody complains :)